lib/change_health/models/response_data.rb

```diff
@@ -0,0 +1,41 @@
+module ChangeHealth
+  module Models
+    module ResponseData
+      attr_reader :response, :raw
+
+      def initialize(data: nil, response: nil)
```

Comment:
Is it bad practice to use the `initialize` method of module?

Reply:
I'm not sure I've ever seen it before but I don't look at modules code as often. I guess I would ask why it's a module when it's acting like a class (having initialize and expecting state). Maybe it's not really a mix-in?

Reply:
I'm going to think about making it a class. I try to prefer composition over inheritance but this might be a good case for it

Reply:
@vechema I agree - maybe there is no inheritence but just other classes include it as a member/association?